src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,71 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Nested
+    @DisplayName("요구사항 1")
+    class 요구사항1 {
+
+        @Test
+        @DisplayName("성공 - split이 된다.")
+        void test1() {
+            String[] result = "1,2".split(",");
+            assertThat(result).containsExactly("1", "2");
```

Comment:
```suggestion
            assertThat("1,2".split(",")).containsExactly("1", "2");
```

위와 같이 한 라인으로 테스트한다면 test2()와 같은 test case로 분리하지 않고도 한 test case에서 검증해도 되지 않을까?